hail_scripts/v01/utils/add_clinvar.py

```diff
@@ -50,17 +50,24 @@
 )"""
 
 
+CLINVAR_VDS_CACHE = { "37": None, "38": None }
+
```

Comment:
I think the keys in this are tuples once its loaded so its a bit confusing that its initialized with these string keys, I think it would be better just tp start with an empty dict

Reply:
oops.. good point. I'll just do = {}.